test/configCases/json/flag-dep-export-perf/index.js

```diff
@@ -0,0 +1,11 @@
+export * from './data.json';
+
+it("should compile and run", () => {
+	expect(__webpack_exports_info__.depth_1.provideInfo).toBe(true)
+	expect(__webpack_exports_info__._depth_1.provideInfo).toBe(true)
+	expect(__webpack_exports_info__.__depth_1.provideInfo).toBe(true)
+
+	expect(__webpack_exports_info__.depth_1.depth_2.provideInfo).toBe(undefined)
+	expect(__webpack_exports_info__._depth_1._depth_2._depth_3._depth_4.provideInfo).toBe(undefined)
+	expect(__webpack_exports_info__.__depth_1[0].__depth_3[0].__depth_5.provideInfo).toBe(undefined)
+});
```

Comment:
Also can you add more tests (not here, just create another one) - we need to make sure that webpack allows us to use deep export when we have `1` (optimizations in this case should not happen)

Reply:
I gotchu.

Reply:
I will also add `module.parser.json.exportsDepth` to the doc later.